examples/test_vault.ipynb

```diff
@@ -0,0 +1,266 @@
+{
```

Comment:
<p>Since the function is very simple and short, I think it will help readability to simply put it here and not redirecting to another file</p><p>(wrap it with <code># nuclio: start-code</code>  and  <code># nuclio: end-code</code>  )</p><p>The <code>context.get_secret</code>  is like the copy-paste source users will look for, though you do give that in your one line example</p><p>again a matter of opinion</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/500/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
I restructured the notebook so that it takes an embedded code, and has different steps in it. Also, I've renamed it - I figured that `test_vault.ipynb` is not a good name. I've renamed it to `mlrun_vault.ipynb`.